includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php

```diff
@@ -285,15 +285,142 @@ private function transform_ece_address_state_data( $address ) {
 			return $address;
 		}
 
-		// states from Apple Pay or Google Pay might be in long format, we need their short format.
+		// Due to a bug in Apple Pay, the "Region" part of a Hong Kong address is delivered in
```

Comment:
I moved this from `includes/express-checkout/class-wc-payments-express-checkout-button-helper.php`, since it was the only place it was used.